odoo/addons/base/views/res_users_views.xml

```diff
@@ -38,7 +38,7 @@
             <field name="arch" type="xml">
                 <!-- the user list is editable, but one cannot add or delete rows -->
                 <tree string="Users" editable="bottom" create="false" delete="false">
-                    <field name="user_id" invisible="1"/>
+                    <field name="user_id" column_invisible="1"/>
```

Comment:
Hello, not why removing completely the field from the view ?

Reply:
I have not clue xD
I updated that in the next commit